nixos/tests/gnome3.nix

```diff
@@ -38,27 +38,36 @@ import ./make-test.nix ({ pkgs, ...} : {
     wmClass = "${gdbus} ${eval} global.display.focus_window.wm_class";
   in ''
       # wait for gdm to start
-      $machine->waitForUnit("display-manager.service");
+      machine.wait_for_unit("display-manager.service")
+
+      machine.sleep(5)
 
       # wait for alice to be logged in
-      $machine->waitForUnit("default.target","alice");
+      machine.wait_for_unit("default.target", "alice")
 
       # Check that logging in has given the user ownership of devices.
-      $machine->succeed("getfacl -p /dev/snd/timer | grep -q alice");
+      machine.succeed("getfacl -p /dev/snd/timer | grep -q alice")
```

Comment:
how about `assert "alice" in machine.succeed("...")`

Reply:
@tfc done.